src/System.Management.Automation/CoreCLR/CorePsPlatform.cs

```diff
@@ -554,27 +554,285 @@ internal static int NonWindowsGetProcessParentPid(int pid)
             return IsMacOS ? Unix.NativeMethods.GetPPid(pid) : Unix.GetProcFSParentPid(pid);
```

Comment:
@JamesWTruher, your last commit had 1 failures in `PowerShell-CI-macos`
Get-FormatData.Can get format data requiring '-PowerShellVersion 5.1'
```powershell
Expected a collection with size 4, but got collection with size 5 @(System.Management.Automation.FormatViewDefinition, System.Management.Automation.FormatViewDefinition, System.Management.Automation.FormatViewDefinition, System.Management.Automation.FormatViewDefinition, System.Management.Automation.FormatViewDefinition).
at <ScriptBlock>, /Users/runner/runners/2.160.0/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/Get-FormatData.Tests.ps1: line 18
18:         $format.FormatViewDefinition | Should -HaveCount 4
```